Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_006.GitHubPat.sarif

```diff
@@ -48,7 +48,6 @@
         {
           "ruleId": "SEC101/006",
           "ruleIndex": 0,
-          "level": "note",
```

Comment:
>note [](http://example.com/codeflow?start=20&length=4)

This delta makes sense. Note disappears, meaning that we now have a warning (by default). A rule with a dynamic validator should return a warning when dynamic analysis is not enabled.